integration_tests/dataproc_test_case.py

```diff
@@ -82,13 +82,9 @@ def tearDown(self):
     def getClusterName(self):
         return self.name
 
-    def upload_test_file(self, name):
+    def upload_test_file(self, testfile, name):
         cmd = 'gcloud compute scp {} {}:'.format(
-            os.path.join(
-                os.path.dirname(os.path.abspath(__file__)),
-                self.COMPONENT.lower(),
-                self.TEST_SCRIPT_FILE_NAME
-            ),
+            testfile,
```

Comment:
If `TEST_SCRIPT_FILE_NAME` is no longer used in the base class, it should be removed from it

Reply:
ok, I will also modify other methods that are using this variable.